frontend/src/metabase/api/user.ts

```diff
@@ -50,6 +51,13 @@ export const userApi = Api.injectEndpoints({
         url: "/api/user/current",
       }),
       providesTags: user => (user ? provideUserTags(user) : []),
+      onCacheEntryAdded: async (_, { dispatch, cacheDataLoaded }) => {
```

Comment:
this is basically an `afterFetchComplete` callback

Reply:
It feels really weird to me that we're maintaining the state in 2 separate areas